events/wc_sacrifice_events.txt

```diff
@@ -0,0 +1,322 @@
+namespace = WCSCRFC
+
+character_event = { # Army commander sacrifice province population
+	id = WCSCRFC.1
+
+	desc = {
+		trigger = { publicly_worship_one_of_old_gods_trigger = yes }
+		text = EVTDESC_WCSCRFC_1_V1
+	}
+	desc = {
+		trigger = { publicly_worship_one_of_old_gods_trigger = no }
+		text = EVTDESC_WCSCRFC_1_V2
+	}
+
+	picture = {
+		trigger = { religion = cult_of_hakkar }
+		picture = GFX_evt_hakkar
+	}
+	picture = {
+		trigger = { religion = old_gods_worship has_character_modifier = follower_of_cthun }
+		picture = GFX_evt_cthun
+	}
+	picture = {
+		trigger = { religion = old_gods_worship has_character_modifier = follower_of_nzoth }
+		picture = GFX_evt_nzoth
+	}
+	picture = {
+		trigger = { religion = old_gods_worship has_character_modifier = follower_of_yogg_saron }
+		picture = GFX_evt_yogg_saron
+	}
+	picture = {
+		trigger = { religion = old_gods_worship has_character_modifier = follower_of_yshaarj }
+		picture = GFX_evt_yshaarj
+	}
+	picture = {
+		trigger = {
+			religion = old_gods_worship
+			NOT = {
+				has_character_modifier = follower_of_cthun
+				has_character_modifier = follower_of_nzoth
+				has_character_modifier = follower_of_yogg_saron
+				has_character_modifier = follower_of_yshaarj
+			}
+		}
+		picture = GFX_evt_voidwalker
+	}
+	picture = {
+		trigger = { NOT = { religion = cult_of_hakkar religion = old_gods_worship  } }
+		picture = GFX_evt_pagan_blood_sacrifice
+	}
+
+	notification = yes
+	is_triggered_only = yes
+
+	trigger = {
+		is_ruler = yes
+		religion_have_sacrifices_trigger = yes
+		FROM = {
+			OR = {
+				holding_type = city
+				holding_type = temple
+				holding_type = castle
+				holding_type = tribal
+			}
+			holder_scope = { NOT = { character = ROOT same_realm = ROOT } }
+			location = {
+				NOT = {
+					has_province_modifier = bloodthirsty_flower_war_province
+					has_province_modifier = depopulated_1
+					has_province_modifier = depopulated_2
+					has_province_modifier = depopulated_3
+				}
+			}
```

Comment:
So, if I get it right, you get offmap points only for the first holding siege.

So if you want to get offmap points, there's no need in sieging more than one holding.
If you want to get gold and piety, there's.

So, it overcomplicates the raiding. And if the player can understand it and chose the right tactic, the AI isn't able to do it. It picks a province and siege holdings, one by one.

Reply:
> I think it's kinda weird when the ruler who gave an order to loot a province not getting points.

Changed

> So if you want to get offmap points, there's no need in sieging more than one holding.

Changed